django/thunderstore/community/templates/community/includes/package_tabs.html

```diff
@@ -1,15 +1,17 @@
 <ul class="nav nav-tabs nav-secondary px-2 pt-2" role="tablist">
     {% for tab in tabs %}
-    <li class="nav-item" role="presentation">
-        <a
-            class="nav-link {% if tab.is_disabled %}disabled text-muted{% else %}clickable{% endif %} {% if tab.is_active %}active{% endif %}"
-            style="user-select: none;"
-            {% if tab.is_active %}aria-selected=true{% endif %}
-            role="tab"
-            {% if not tab.is_disabled %}href="{{ tab.url }}"{% endif %}
-        >
-            {{ tab.title }}
-        </a>
-    </li>
+        {% if tab.is_visible %}
+            <li class="nav-item" role="presentation">
+                <a
+                    class="nav-link {% if tab.is_disabled %}disabled text-muted{% else %}clickable{% endif %} {% if tab.is_active %}active{% endif %}"
+                    style="user-select: none;"
+                    {% if tab.is_active %}aria-selected=true{% endif %}
+                    role="tab"
+                    {% if not tab.is_disabled %}href="{{ tab.url }}"{% endif %}
+                >
+                    {{ tab.title }}
+                </a>
+            </li>
+        {% endif %}
```

Comment:
Personally I would applied the filtering in the `get_tab_context` in `PackageTabsMixin` rather than the template here as it would keep the template simpler, but this is good enough to me as well if you want to move forward w/o more changes.

More specifically, amending the loop at the return statement to be roughly
```python
Tab(...) for k, v in tabs.items() if v.is_visible
```
would have done the trick